cmd/mesh/rpc_handler.go

```diff
@@ -86,8 +110,19 @@ func (handler *rpcHandler) AddOrders(signedOrdersRaw []*json.RawMessage, opts rp
 }
 
 // AddPeer is called when an RPC client calls AddPeer,
-func (handler *rpcHandler) AddPeer(peerInfo peerstore.PeerInfo) error {
+func (handler *rpcHandler) AddPeer(peerInfo peerstore.PeerInfo) (errRes error) {
```

Comment:
Convention is to just call this `err`.

```suggestion
func (handler *rpcHandler) AddPeer(peerInfo peerstore.PeerInfo) (err error) {
```